src/basic/time-util.h

```diff
@@ -29,6 +29,15 @@ typedef struct triple_timestamp {
         usec_t boottime;
 } triple_timestamp;
 
+typedef enum TimestampStyle {
+        TIMESTAMP_PRETTY,
+        TIMESTAMP_US,
+        TIMESTAMP_UTC,
+        TIMESTAMP_US_UTC,
```

Comment:
Reading those three enum values next to each other it is not clear that `_US` stands for microseconds and not a (imaginary) US timezone (in opposition to UTC time). I would rename them:
```
TIMESTAMP_PRETTY,
TIMESTAMP_UTC, /* seconds since epoch with integer resolution */
TIMESTAMP_UTC_MICROSECONDS, /* seconds since epoch with microseconds resolution */
TIMESTAMP_MICROSECONDS, /* what does this actually do? */
```

Reply:
Uh! I read the formatting code and it seems that the enum names are more more confusing than what I though.

Reply:
Renamed and added comment that shows the format

Reply:
Thanks! It is still confusing, but less so now. It would probably make sense (also for how the enum values are used) to have `TIMESTAMP_FORMAT_UTC` and `TIMESTAMP_FORMAT_MICROSEC` as flags rather than members of an enum so that only two flags are used instead than four distinct enum values encoding all permutations. But I don't want to bikeshed this too much :-)